go/vt/vttest/environment.go

```diff
@@ -92,6 +92,9 @@ type Environment interface {
 	// any temporary data in the environment. Environments that can
 	// last through several test runs do not need to implement it.
 	TearDown() error
+
+	// IsSQLFlavor retruns the flavor or SQL Server running
```

Comment:
this function is not returning the flavor but true if it is not mariadb .... Maybe rename this function and other such functions in the code. Actually I would first like to see if we can avoid the init_maria_db.sql entirely using conditional code in init_db.sql ...

Reply:
done